src/test/java/seedu/address/testutil/TypicalPersons.java

```diff
@@ -83,6 +83,38 @@ public static AddressBook getTypicalAddressBook() {
         return ab;
     }
 
+    /**
+     * Returns an {@code AddressBook} with typical
+     * persons in unsorted order lexicographically.
+     */
+    public static AddressBook getUnsortedAddressBook() {
```

Comment:
Could we add a person whose first letter is the same as another person? E.g. ALICE, AARON

Reply:
Thank you for the suggestion. I will definitely add it in.